src/workflow/ArcanistLandWorkflow.php

```diff
@@ -1028,7 +1028,23 @@ private function push() {
         'Holding change in **%s**: it has NOT been pushed yet.',
         $this->onto)."\n");
     } else {
-      echo pht('Pushing change...'), "\n\n";
+      $sirmixalot_enrolled = $this->getConfigFromAnySource(
+        'sirmixalot.enrolled',
```

Comment:
I'd prefix our custom settings with `uber.`: `uber.sirmixalot.enrolled`. It will be easier to recognize the parts which were added by us.